MoreCyclopsUpgrades/Managers/CyclopsHUDManager.cs

```diff
@@ -3,16 +3,39 @@
     using Common;
     using MoreCyclopsUpgrades.SaveData;
     using UnityEngine;
+    using UnityEngine.UI;
 
     internal class CyclopsHUDManager
     {
+        private class Indicator
```

Comment:
Add some utility method to this class to handle converting numbers to string text.  
This method would handle converting large numbers into more readable ones, and could possibly also handle other formatting.